### PR TITLE
Pioneer 13 Glider updates

### PR DESCRIPTION
**Pull request by cdobs**
These are the asset management updates pertaining to gliders on the P13 cruise and include changes to the following files: 
-cp_340/cp_379 deployment csvs
-new calibration csvs
-updates to the sensor bulkload

**Sensor info:**
340:
-CTD SN 9029: Already in bulkload. No new calibrations.
-FLORTM SN 3921: Already in bulkload. No new calibrations.
-DOSTAM SN 398: Already in bulkload. No previous calibrations in the system. Added new calibration csv for 4/12/2019 cal.
-PARADM SN 50145: Already in bulkload. No new calibrations.
-ADCPAM SN 545/648878: Was not in the bulkload as xducer number, so that was added (line 402). Also, no calibration CSVs in the system for the ducer number so I created one using the date for the SN 545 cal csv.

379:
-CTD SN 9347: Was already in bulkload. New calibration csv (10/16/2018) added.
-FLORTM SN 4201: Was not in bulk load, we have never deployed this sensor even though it is old and not freshly calibrated. Added to bulkload (line 1439). New calibration csv (1/11/2016) added.
-DOSTAM SN 0142: Already in bulk load. No new calibrations.
-PARADM SN 50304: New par sensor. Added to bulk load (line 2062). New calibration csv (7/15/2019) added.
-ADCPAM SN 548/650865: xducer SN already in bulkload. There were no existing calibration CSVs for the xducer SN. I created two, using the same dates of calibration CSVs for DVL bay SN 548 that were already in the system. This bay has been deployed with this xducer since we purchased it.

### CODE REVIEW

File: calibration/ADCPAM/CGINS-ADCPAM-48878__20181020.csv [hunk 1]
```diff
@@ -0,0 +1,5 @@
+serial,name,value,notes
+48878,CC_scale_factor1,0.61,DVL Bay 0545; FCP 3304-00150-00111.docs
+48878,CC_scale_factor2,0.61,
+48878,CC_scale_factor3,0.61,
+48878,CC_scale_factor4,0.61,
```
Comment (snwhite) [inline calibration/ADCPAM/CGINS-ADCPAM-48878__20181020.csv]: Looks good (SNW) – I added DVL Bay S/N and FCP file.

File: calibration/ADCPAM/CGINS-ADCPAM-50865__20130802.csv [hunk 2]
```diff
@@ -0,0 +1,5 @@
+serial,name,value,notes
+50865,CC_scale_factor1,0.61,DVL Bay 0548; FCP 3304-00150-00112.docx
+50865,CC_scale_factor2,0.61,
+50865,CC_scale_factor3,0.61,
+50865,CC_scale_factor4,0.61,
```
Comment (snwhite) [inline calibration/ADCPAM/CGINS-ADCPAM-50865__20130802.csv]: Looks good (SNW) – I added DVL Bay S/N and FCP file.

File: calibration/ADCPAM/CGINS-ADCPAM-50865__20140131.csv [hunk 3]
```diff
@@ -0,0 +1,5 @@
+serial,name,value,notes
+50865,CC_scale_factor1,0.61,DVL Bay 0548; FCP 3304-00150-00112.docx
+50865,CC_scale_factor2,0.61,
+50865,CC_scale_factor3,0.61,
+50865,CC_scale_factor4,0.61,
```
Comment (snwhite) [inline calibration/ADCPAM/CGINS-ADCPAM-50865__20140131.csv]: Looks good (SNW) – I added DVL Bay S/N and FCP file.

File: calibration/CTDGVM/CGINS-CTDGVM-09347__20181016.csv [hunk 4]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+9347,CC_calibration_date,1539648000000,sensor calibration date (milliseconds since 1970-01-01)
```
Comment (snwhite) [inline calibration/CTDGVM/CGINS-CTDGVM-09347__20181016.csv]: Looks good (SNW)

File: calibration/DOSTAM/CGINS-DOSTAM-00398__20190412.csv [hunk 5]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+398,CC_calibration_date,1555027200000,sensor calibration date (milliseconds since 1970-01-01)
```
Comment (snwhite) [inline calibration/DOSTAM/CGINS-DOSTAM-00398__20190412.csv]: Looks good (SNW)

File: calibration/FLORTM/CGINS-FLORTM-04201__20160111.csv [hunk 6]
```diff
@@ -0,0 +1,5 @@
+serial,name,value,notes
+4201,CC_angular_resolution,1.076,
+4201,CC_depolarization_ratio,0.039,
+4201,CC_measurement_wavelength,700.0,
+4201,CC_scattering_angle,124.0,
```
Comment (snwhite) [inline calibration/FLORTM/CGINS-FLORTM-04201__20160111.csv]: Looks good (SNW)

File: calibration/PARADM/CGINS-PARADM-50304__20190715.csv [hunk 7]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+50304,CC_bsipar_par_scaling,1,constant
```
Comment (snwhite) [inline calibration/PARADM/CGINS-PARADM-50304__20190715.csv]: Looks good (SNW)

File: deployment/CP05MOAS-GL340_Deploy.csv [hunk 8]
```diff
@@ -46,4 +46,10 @@ TS20190205,,BK20190504,,CP05MOAS-GL340-02-FLORTM000,8,1,2019-02-05T08:24:00,2019
...
\ No newline at end of file
+TS20190205,,BK20190504,,CP05MOAS-GL340-01-ADCPAM000,8,1,2019-02-05T08:24:00,2019-05-04T14:00:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00340,CGINS-ADCPAM-00537,39.91563333,-70.36722222,,200,200,EB 
+AR-39,,,,CP05MOAS-GL340-00-ENG000000,9,1,2019-09-25T00:00:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00340,CGCON-GLDRCP-00340,,,,200,200,EB 
+AR-39,,,,CP05MOAS-GL340-02-FLORTM000,9,1,2019-09-25T00:00:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00340,CGINS-FLORTM-03921,,,,200,200,EB 
+AR-39,,,,CP05MOAS-GL340-03-CTDGVM000,9,1,2019-09-25T00:00:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00340,CGINS-CTDGVM-09029,,,,200,200,EB 
+AR-39,,,,CP05MOAS-GL340-04-DOSTAM000,9,1,2019-09-25T00:00:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00340,CGINS-DOSTAM-00398,,,,200,200,EB 
+AR-39,,,,CP05MOAS-GL340-05-PARADM000,9,1,2019-09-25T00:00:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00340,CGINS-PARADM-50145,,,,200,200,EB 
+AR-39,,,,CP05MOAS-GL340-01-ADCPAM000,9,1,2019-09-25T00:00:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00340,CGINS-ADCPAM-48878,,,,200,200,EB
```
Comment (snwhite) [inline deployment/CP05MOAS-GL340_Deploy.csv]: Looks good (SNW)

File: deployment/CP05MOAS-GL379_Deploy.csv [hunk 9]
```diff
@@ -35,9 +35,15 @@ TS20190205,,TS20190516,,CP05MOAS-GL379-02-FLORTM000,6,1,2019-02-05T07:54:00,2019
...
+WJ20190617,,,,CP05MOAS-GL379-00-ENG000000,7,1,2019-06-18T23:14:29,2019-07-28T02:39:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGCON-GLDRCP-00379,39.9156,-70.9142,,1000,1000,FZ-1
+WJ20190617,,,,CP05MOAS-GL379-01-ADCPAM000,7,1,2019-06-18T23:14:29,2019-07-28T02:39:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-ADCPAM-49982,39.9156,-70.9142,,1000,1000,FZ-1 Will change UID to CGINS-ADCPAM-00548
+WJ20190617,,,,CP05MOAS-GL379-02-FLORTM000,7,1,2019-06-18T23:14:29,2019-07-28T02:39:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-FLORTM-05133,39.9156,-70.9142,,1000,1000,FZ-1
+WJ20190617,,,,CP05MOAS-GL379-03-CTDGVM000,7,1,2019-06-18T23:14:29,2019-07-28T02:39:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-CTDGVM-09453,39.9156,-70.9142,,1000,1000,FZ-1
+WJ20190617,,,,CP05MOAS-GL379-04-DOSTAM000,7,1,2019-06-18T23:14:29,2019-07-28T02:39:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-DOSTAM-00142,39.9156,-70.9142,,1000,1000,FZ-1
+WJ20190617,,,,CP05MOAS-GL379-05-PARADM000,7,1,2019-06-18T23:14:29,2019-07-28T02:39:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-PARADM-50277,39.9156,-70.9142,,1000,1000,FZ-1
```
Comment (snwhite) [inline deployment/CP05MOAS-GL379_Deploy.csv]: Looks good, but should they be a recovery cruise #?

File: bulk/sensor_bulk_load-AssetRecord.csv [hunk 10]
```diff
@@ -1,2577 +1,2581 @@
...
-ATOSU-58332-00008,ATOSU-58332-00008,Sensor,0,Spectrophotometer: OPTAA Series D,OPTAA SERIES D,WET Labs,AC-S,ACS-221,,20150501,40133.75,,
-ATOSU-58332-00009,ATOSU-58332-00009,Sensor,0,Spectrophotometer: OPTAA Series D,OPTAA-D,WET Labs,AC-S,ACS-169,,,,,
-ATOSU-58336-00005,ATOSU-58336-00005,Sensor,0,pCO2 Water: PCO2W Series A,PCO2W SERIES A (SAMI-CO2),Sunburst Sensors,SAMI-pCO2,C0078,,20140109,17942.3,,
-ATOSU-58336-00006,ATOSU-58336-00006,Sensor,0,pCO2 Water: PCO2W Series A,PCO2W SERIES A (SAMI-CO2),Sunburst Sensors,SAMI-pCO2,C0079,,20140127,17942.3,,
-ATOSU-58336-00007,ATOSU-58336-00007,Sensor,0,pCO2 Water: PCO2W Series A,pCO2 sensor PCO2W-A,Sunburst Sensors,SAMI-pCO2,C0116,,,18054,,
-ATOSU-58336-00008,ATOSU-58336-00008,Sensor,0,pCO2 Water: PCO2W Series A,pCO2 sensor PCO2W-A,Sunburst Sensors,SAMI-pCO2,C0117,,,18544.5,,
-ATOSU-58337-00007,ATOSU-58337-00007,Sensor,0,Seawater pH: PHSEN Series A,PHSEN SERIES D (SAMI-PH),Sunburst Sensors,SAMI-pH,P0115,,20140109,17479.95,,
-ATOSU-58337-00008,ATOSU-58337-00008,Sensor,0,Seawater pH: PHSEN Series A,PHSEN SERIES D (SAMI-PH),Sunburst Sensors,SAMI-pH,P0131,,20140425,18003.4,,
-ATOSU-5
```
[the remark this answers is not in the document]
Comment (rtravis-whoi) [reply bulk/sensor_bulk_load-AssetRecord.csv]: Extra commas at end of line 402? ADCPAM-48878

File: calibration/ADCPAM/CGINS-ADCPAM-48878__20181020.csv (hunk 1 above)
Comment (rtravis-whoi) [reply calibration/ADCPAM/CGINS-ADCPAM-48878__20181020.csv]: Looks good RGT

File: calibration/ADCPAM/CGINS-ADCPAM-50865__20130802.csv (hunk 2 above)
Comment (rtravis-whoi) [reply calibration/ADCPAM/CGINS-ADCPAM-50865__20130802.csv]: Would this earlier one still refer to same FCP?  (RGT)

File: calibration/ADCPAM/CGINS-ADCPAM-50865__20140131.csv (hunk 3 above)
Comment (rtravis-whoi) [reply calibration/ADCPAM/CGINS-ADCPAM-50865__20140131.csv]: Looks good RGT

File: calibration/CTDGVM/CGINS-CTDGVM-09347__20181016.csv (hunk 4 above)
Comment (rtravis-whoi) [reply calibration/CTDGVM/CGINS-CTDGVM-09347__20181016.csv]: looks good

File: calibration/DOSTAM/CGINS-DOSTAM-00398__20190412.csv (hunk 5 above)
Comment (rtravis-whoi) [reply calibration/DOSTAM/CGINS-DOSTAM-00398__20190412.csv]: looks good

File: calibration/FLORTM/CGINS-FLORTM-04201__20160111.csv (hunk 6 above)
Comment (rtravis-whoi) [reply calibration/FLORTM/CGINS-FLORTM-04201__20160111.csv]: looks good

File: calibration/PARADM/CGINS-PARADM-50304__20190715.csv (hunk 7 above)
Comment (rtravis-whoi) [reply calibration/PARADM/CGINS-PARADM-50304__20190715.csv]: looks good

File: deployment/CP05MOAS-GL340_Deploy.csv (hunk 8 above)
Comment (rtravis-whoi) [reply deployment/CP05MOAS-GL340_Deploy.csv]: looks good

File: deployment/CP05MOAS-GL379_Deploy.csv (hunk 9 above)
Comment (rtravis-whoi) [reply deployment/CP05MOAS-GL379_Deploy.csv]: looks good for P13 deploy; there should be a recover cruise # but it won't 'break' anything if we don't have it in there now.  Could updated after P13 deployments are complete...

**Pull request merged**